ml_genn/model.py

```diff
@@ -179,35 +179,35 @@ def evaluate_batched(self, data, time, save_samples=[]):
         spike_t       --  list of spike times for each sample index in save_samples
         """
 
+        n_complete = [0] * len(self.outputs)
```

Comment:
I think it would be good to check that the number of inputs and outputs in the model match the data passed to the function